Solutions/Endjin.Imm.Domain/Repository/RuleDefinitionRepository.cs

```diff
@@ -16,7 +16,7 @@ public RuleDefinitionRepository(IpMaturityMatrixRuleset ruleSet)
             this.ruleSet = ruleSet;
         }
 
-        public RuleDefinition GetDefinitionFor(Rule rule)
+        public RuleDefinition GetDefinitionFor(RuleAssertion rule)
```

Comment:
Should the parameter name be updated too?

Reply:
Fixed - thanks!